css/properties/mask-size.json

```
@@ -27,24 +29,27 @@
               "version_added": false
             },
             "opera": {
-              "version_added": false
+              "prefix": "-webkit-",
+              "version_added": "15"
             },
             "opera_android": {
-              "version_added": null
+              "prefix": "-webkit-",
+              "version_added": "15"
```

Perhaps also comes from the typo above?

```suggestion
              "version_added": "14"
```